ckan/templates/snippets/organization_changes/description.html

```diff
@@ -0,0 +1,42 @@
+<li>
+  <p>
+    {% if change.method == "change" %}
+
+      {{ _('Updated description of {pkg_link} from <br class="line-height2"><blockquote>{old_description}</blockquote> to <br class="line-height2"><blockquote>{new_description}</blockquote>').format(
+              pkg_link = '<a href="{pkg_url}">{pkg_name}</a>'.format(
+                pkg_url = h.url_for(controller='organization',
+                                    action='read', id=change.pkg_id),
+                pkg_name = change.title
+              )|safe,
+              old_description = change.old_description,
+              new_description = change.new_description
+        ) }}
+
+    {% elif change.method == "add" %}
+
+      {{ _('Updated description of {pkg_link} to <br class="line-height2"><blockquote>{new_description}</blockquote>').format(
+              pkg_link = '<a href="{pkg_url}">{pkg_name}</a>'.format(
+                pkg_url = h.url_for(controller='organization',
```

Comment:
Should be url_for to named route instead on controller/action as controllers are soon removed. See https://github.com/ckan/ckan/blob/abfda8b401ec370c9c74fc67678dcb38f8c72a26/ckan/templates/snippets/changes/notes.html#L14